js/package.json

```diff
@@ -27,6 +27,7 @@
     "css-loader": "^3.0.0",
     "file-loader": "^4.0.0",
     "npm-run-all": "^4.1.5",
+    "promise-semaphore": "^0.2.8",
```

Comment:
This package is still using bluebird promises and hasn't been updated in 3 years.

Reply:
Yes, I'm not happy with that package either, looking for something else.  https://github.com/sindresorhus/p-queue may actually do all of this..